content/learning-paths/cross-platform/eigen-linear-algebra-on-arm/eigen-intro.md

```diff
@@ -25,37 +25,38 @@ Current implementations include:
 
 It has also been ported to SYCL and CUDA backends and more ports are in progress.
 
-## Where is it used
+## Where is Eigen used?
 
-Eigen is a very useful library and its usefulness is proven by the number of projects that are using it. 
-[The list of projects that are using Eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page#Projects_using_Eigen) is rich but here are a few high profile ones:
+Eigen is a useful library and its success is proven by the number of projects that use it. 
+[The list of projects using Eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page#Projects_using_Eigen) is rich, but here are a few high profile projects:
 
-* [Tensorflow](https://www.tensorflow.org/) - An open source software library for Machine Intelligence
+* [Tensorflow](https://www.tensorflow.org/) - An open-source software library for Machine Intelligence
```

Comment:
This should be left "open source" as per the definition, somehow the dash got into that, but it's incorrect. The correct term is without the dash.